messages.js

```diff
@@ -0,0 +1,44 @@
+const helpMsg = '\n\n' +
+    '**TunnelQuestBot Help**\n' +
+    '***NOTE:***\n' +
+    ' • All commands begin with an exclamation mark (\"!\").\n' +
+    ' • Arguments listed in `code blocks` should be replaced with your input data.\n' +
+    ' • Commas *are* actually required.\n' +
+    ' • Item names are not case sensitive.\n' +
+    ' • You may enter prices in pp or k (ex: 1100pp or 1.1k).\n' +
+    ' • Parser will not detect aliases (ex: watching "Thick Banded Belt" will not detect "TBB"), however this is a future goal.\n' +
+    '\n' +
+    '***COMMANDS:***\n' +
+    '!help' +
+    '> Displays available commands.\n' +
+    '!add watch: `item`, `maximum price`, `server`' +
+    '> Starts a watch based on entered parameters - watches expire after 7 days.  Price is optional.\n' +
+    '!end watch: `item`, `server`' +
+    '> Ends a currently running watch.\n' +
+    '!end all watches' +
+    '> Ends all currently running watches.\n' +
+    '!extend all watches' +
+    '> Extends your current watches another 7 days.\n' +
+    '!show watch: `item`, `server`' +
+    '> Lists details for a watch for entered item - if no arguments are provided, behaves as *!show watches*.\n' +
+    '!show watches' +
+    '> Lists details for all watches.\n' +
+    '\n' +
+    '***TIPS:***\n' +
+    ' • You can use `!add watch` to update an existing watch if you wish to modify the price and/or reset the 7 day expiration timer.\n' +
+    ' • To report a problem or request a feature, talk to us in #feedback_and_ideas, or create an issue here: https://github.com/jamesjamail/TunnelQuestBot/issues'
+
+
+const welcomeMsg = 'I am TunnelQuestBot, your helpful gnome assistant. Please allow me to make buying and selling items easier so you can finally start tipping for ports.\n\n' +
+    'I watch EC auctions on both Blue and Green servers. If you\'re in the market for a new sword, you can enter the following command as a direct message to me, or in the \`public_command_space channel\`:\n\n' +
```

Comment:
Maybe put "channel" outside of the code block? and make it #public_command_space (I assume we can't link to it via channel-id in a PM? did you try? maybe it randomly works lol)

Reply:
What's the functionality you are aiming for?

Reply:
I was just curious if it's possible to make a clickable channel link to a server, in a DM. Not sure it's possible.

Reply:
regardless, at least changing:
as a direct message to me, or in the \`public_command_space channel\`:\n\n'
to:
as a direct message to me, or in the \`#public_command_space\` channel:\n\n'
would be good I think?

Reply:
You can make a clickable link to a channel from a message.  I'm down to refactor to that eventually, though I'd rather integrate this PR the way it is now because I don't have much bandwidth today.

I'll add the '#' though!